room-scene/room.js

```diff
@@ -39,19 +46,29 @@ function toggleKettle() {
 function toggleTv() {
     if (state.isTvOn == true) {
         state.isTvOn = false;
-        $("#tv").css("background-image:", "");
```

Comment:
channels could be classes. eg. 
.tv_off {"background-color:", "black"}
.tv_channel_123 {"background-image:", "nyan.gif"}
...
state.isTvOn = !state.isTvOn;
if (state.isTvOn) {
...
} else {
  $("#tv")
    .removeClass('tv_channel_123')
    .removeClass('tv_channel_345')
    .addClass('tv_off');
}